docs/PRIVACY.md

```diff
@@ -0,0 +1,53 @@
+# WHO COVID-19 App Privacy Notice
+
+This App Privacy Notice supplements the general World Health Organization (“WHO” or “us”, “we” or “our”) Privacy Policy available at [https://www.who.int/about/who-we-are/privacy-policy][1] (the “WHO Privacy Policy”), and applies to mobile applications and services made available by the WHO (each an “App” or “Service”).
+
+By using a Service, you agree to the collection and use of data in accordance with these policies. We do not disclose, share, sell, or transfer any information about Service users unless required by law or for law enforcement.
+
+## Data We Collect
+
+When you use our Service we:
+
+* Do not collect data that identifies you personally (“Personal Data”) unless you choose to provide that information to us.
+* Do collect information that does not identify you personally, including data collected automatically either generated by the use of the Service or from the Service infrastructure (“Usage Data”). Usage Data may include location information, which we may use to provide you with more relevant information or language choices.
+
+Our Apps are not intended to be used by or to solicit information of any kind from children under age 13.
+
+## How We Use Data
+
+The use of data described in the WHO Privacy Policy applies to the Service. Additionally, we use Usage Data to enable participation in interactive features of our Service and to provide you with information if you opt into receiving information through the Service or engage with interactive features.
+
+## Third-Party Apps and Services
+
+Third-party applications and/or services linked to or embedded in the App governed by those third-party security and privacy policies (see [third party tools][8]). If you consent to receiving push notifications, a third-party provider may collect device and regional location information, but your identity or Personal Data will not be revealed to us as a result of enabling such notifications. You may stop push notifications by adjusting your device settings.
```

Comment:
Content mismatch: "third party provider" doesn't have a dash in the [current privacy.pdf](https://github.com/WorldHealthOrganization/app/blob/069bfe9/server/appengine/src/main/webapp/privacy.pdf)